apps/webhook_listeners/lib/Controller/WebhooksController.php

```diff
@@ -48,16 +48,21 @@ public function __construct(
 	/**
 	 * List registered webhooks
 	 *
+	 * @param string|null $uri The callback URI to filter by (optional)
```

Comment:
```suggestion
	 * @param string|null $uri The callback URI to filter by
```
Not necessary as the nullability already indicates this.